components/scream/scripts/utils.py

```diff
@@ -350,6 +350,9 @@ def get_current_head(repo=None):
 def is_repo_clean(repo=None):
 ###############################################################################
     rc, output, _ = run_cmd("git status --porcelain --untracked-files=no", combine_output=True, from_dir=repo)
+    if rc != 0 or output != "":
+        print("Warning: repo is not clean: {}".format(output))
```

Comment:
Uhm, if we go this route, we should add a verbose flag to is_repo_clean, cause it's also called in `cleanup_repo`, and we don't want that to print messages. We only want the print when `is_repo_clean` is used inside `expect` clauses.

Reply:
I think we almost always want to know the details if the repo is being left in a dirty state. The only case where we expect dirty files is if the user provided `--keep-tree`. I will add a silent flag for that call.